mesonbuild/compilers/c.py

```diff
@@ -1176,7 +1176,7 @@ def get_options(self):
                                                        ['none', 'c89', 'c99', 'c11',
                                                         'gnu89', 'gnu99', 'gnu11'],
                                                        'none')})
-        if self.compiler_type == CompilerType.GCC_MINGW:
+        if self.compiler_type.is_windows_compiler:
```

Comment:
This should be `is_windows_compiler()` so the function is called? (existing uses are also defective??)

Reply:
Ah, no. wait. Insufficent coffee.  Those methods have the `@property` decorator, so everything is fine :)